azure-pipelines-docs-integration.yml

```diff
@@ -82,6 +82,7 @@ stages:
       vmImage: 'ubuntu-latest'
     jobs:
     - job: test_docs
+      timeoutInMinutes: 120
```

Comment:
The default being 60, I'm fine with this change :bow:

Reply:
Something crept up in one of the runs and the pipeline was killed at 60 minutes while it was still doing stuff